### PR TITLE
Issue7485-FixFormattingOfIntroParagraphInAuditTopic

### PR DESCRIPTION
**Pull request by emilytee**

### CODE REVIEW

**Comment by emilytee** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties [hunk 1]
```diff
@@ -15,19 +15,33 @@
...
- - Client certificate login \
- - Servlet runAs delegation \
- - Form logout \
- - Servlet 3.0 login, logout and authenticate APIs \
-The Liberty Audit feature supports the Cloud Auditing Data Federation (CADF) event model.  The CADF model describes a data model and associated schema definitions for an audit event. \
-The feature provides a default implementation, the AuditFileHandler, which emits human-readable audit records to a file-based log.   Each audit record is emitted in JSON format. \
-
-
+description=The Liberty Audit feature is used to report and track auditable events to ensure the integrity of your system. \n\
+The Liberty Audit feature introduces an infrastructure which serves two purposes: \n\n\
```
Comment (Charlotte-Holt) [inline dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: Change
"The Liberty Audit feature introduces an infrastructure **which** serves two purposes:"
to
"The Liberty Audit feature introduces an infrastructure **that** serves two purposes:"

File: dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties [hunk 2]
```diff
@@ -15,19 +15,33 @@
...
+ - Confirming the effectiveness and integrity of the existing configuration \n\
+ - Identifying areas where improvement to the configuration may be needed  \n\n\
```
Comment (Charlotte-Holt) [inline dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: Change
"- Identifying areas where improvement to the configuration **may** be needed"
to
"- Identifying areas where improvement to the configuration **might** be needed"

File: dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties [hunk 3]
```diff
@@ -15,19 +15,33 @@
...
+The Liberty Audit feature has the ability to capture the following auditable events: \n\n\
```
Comment (Charlotte-Holt) [inline dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: Change
"The Liberty Audit feature **has the ability** to capture the following auditable events:"
to
"The Liberty Audit feature **is able** to capture the following auditable events:"

File: dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties [hunk 4]
```diff
@@ -15,19 +15,33 @@
...
+ - JMX MBean operations \n\
+ - JMX Notifications \n\
+ - JMX MBean registration \n\
+ - JMX MBean attribute operations \n\
+ - JMS Authentication \n\
+ - JMS Authorization \n\
+ - OAuth application password and token management \n\
+ - SAF authorization \n\n\
+The Liberty Audit feature supports the Cloud Auditing Data Federation (CADF) event model.  The CADF model describes a data model and associated schema definitions for an audit event. \n\n\
+The feature provides a default implementation, the AuditFileHandler, which emits human-readable audit records to a file-based log.   Each audit record is emitted in JSON format.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: Only 1 space between "(CADF) event model." and "The CADF model", and only one space between "file-based log." and "Each audit record"

File: dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties (hunk 1 above)
Comment (emilytee) [reply dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: That all is old text that had been previously reviewed and approved.  I'm not changing any of that.  Just changing how the tab-ing appears, and adding some missed items.

File: dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties (hunk 3 above)
Comment (emilytee) [reply dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: Ditto as above

File: dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties (hunk 4 above)
Comment (emilytee) [reply dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: Overall question:  All of this text was reviewed by IDD a year and half ago, and what is there is based on IDD's feedback then.  If none of the text is being changed, why is it getting re-reviewed?    Different person, different feedback, I get it.   But we're not changing text here, just format.

Comment (Charlotte-Holt) [reply dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties]: @emilytee LibbyBot flagged it as requiring an ID review, so I reviewed what was changed. Sometimes we get a different pair of eyes on something and pick up things that might've been overlooked. All of the changes I suggested are minor.

**Comment by andrea-pichler** [comment]
No message review from L2 required.

**Pull request merged**